src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.ts

```diff
@@ -433,7 +433,7 @@ export class JJCountComponent implements OnInit, OnChanges {
       } else {
         this.form.get('totalFineAmount').setValue(this.jjDisputedCount?.ticketedFineAmount);
         this.form.get('lesserOrGreaterAmount').setValue(this.jjDisputedCount?.ticketedFineAmount);
-        this.bindRevisedDueDate(this.jjDisputedCount.revisedDueDate);
+        this.form.controls.revisedDueDate.setValue(new Date()); // TCVP-3082 & TCVP-3070 implemented this based on multiple scenarios on findings and time to pay selections

```

Comment:
Shouldn't we set the `revisedDueDate` entered by the Staff if found guilty? If we're not setting it here to the value entered by the Staff, then where should that be set? I think this should be set to the `revisedDueDate` if that has been entered by the staff, otherwise set to new Date() if null.